ENIGMAsystem/SHELL/Universal_System/Resources/backgroundinit.cpp

```diff
@@ -41,14 +41,15 @@ namespace enigma
 
     // Determine how many backgrounds we have
     int bkgcount;
-    if (!fread(&bkgcount,4,1,exe))
+    if (!fread(&bkgcount,4,1,exe) || bkgcount == 0)
```

Comment:
As I said on #1979, even when count is 0, it's still possible you have some sprites that were deleted. If you just create a sprite and delete it, the max high id is still incremented and not reset until defragging.

Reply:
that count is just data read off the exe it should never change at runtime